src/components/profile/agent-wallet-info.tsx

```diff
@@ -21,106 +27,131 @@ export function WalletInfoCard({
 }: WalletInfoCardProps) {
   const { copiedText, copyToClipboard } = useClipboard();
 
+  const getAddressExplorerUrl = (address: string) => {
+    const baseUrl = "https://explorer.hiro.so/address";
+    const isTestnet = process.env.NEXT_PUBLIC_STACKS_NETWORK === "testnet";
+    return `${baseUrl}/${address}${isTestnet ? "?chain=testnet" : ""}`;
```

Comment:
fyi you could pass the network directly e.g. `?chain=testnet` and `?chain=mainnet`

mainnet is the default if none is specified